site/viamguides/src/environment-variables/environment-variables.md

```diff
@@ -0,0 +1,223 @@
+### Run the sample code
+1. **Create `.env` file**: From the command line, create a new file called `.env` to store our environment variables to use in local development.
+    ```bash
+    touch .env
+    ```
+1. **Input environment variables**: Add your environment variables to the `.env` file, formatted like the following:
+    ```
+    ROBOT_API_KEY="sijdb24bjnxsmp18mij0hace6smihu0r"
+    ROBOT_API_KEY_ID="9e135013-f3bf-40fc-a0e3-e6fc66a418d7"
+    ```
+1. **Install `python-dotenv` package**: From the command line, still within our virtual environment `.venv`, install the [`python-dotenv`](https://pypi.org/project/python-dotenv/) package to read key-value pairs from an `.env `file and can set them as environment variables.
+    ```bash
+    pip3 install python-dotenv
+   ```
+1. **Load the variables into your environment**: At the top of your `square.py` file, add the following code to load variables into your environment.
+    ```python
+    from dotenv import load_dotenv
+    import os
+
+    load_dotenv()  # Loads the environment variables from the .env file
+
+    robot_api_key = os.getenv('ROBOT_API_KEY')
+    robot_api_key_id = os.getenv('ROBOT_API_KEY_ID')
+    ```
+1. **Use variables in your code**: You can now use these variables within your code. On rows 7 and 8 in `square.py`, you load variables into the environment. And then on rows 20 and 21, you use those variables to access your machine.
+  ![Code in VS Code](assets/vscode.png)
+1. **Run the code:** Run the sample code to connect to your machine
+   ```bash
+   python3 square.py
+   ```
+
+If you want to play with the rover, go to the Viam app **CONTROL** tab to interact with your machine’s resources. Click on one of the camera panels and toggle the camera stream on so you can observe the rover’s movements. Or try adding more code to [drive the rover in a square](https://docs.viam.com/get-started/drive-rover/).
+
+In the next few sections, learn how to use variables on your machine in production.
+
+<!-- ------------------------ -->
+## Add the control code to your machine
+Duration: 3
+
+Once you have the code working in your local development environment, you may choose to move your control code to your machine, so that it can run in production. 
+
+> aside positive
+> If you followed [Try Viam](https://app.viam.com/try), you won't have access to add control code to the machine, since you are just borrowing the rover. However, you can skip to the next part to see how you can set up variables on the machine. If you're using your own wheeled robot, you can complete these steps.
+
+### Add the control code to your machine
+1. **SSH into your board**: From the terminal window, run the following command to [SSH (Secure Shell) into your board](https://docs.viam.com/installation/prepare/rpi-setup/#connect-with-ssh), where the text in `<>` should be replaced (including the `<` and `>` symbols themselves) with the `user` and `hostname` you configured when you set up your machine. 
```

Comment:
On some networks, mDNS might not work as expected so the `hostname.local` alias would fail to resolve. Should we described a fallback to the static IP address shown in the Viam app just in case?